build/index.ts

```diff
@@ -175,6 +176,15 @@ export async function buildDocument(
   document,
   documentOptions: DocumentOptions = {}
 ): Promise<BuiltDocument> {
+  Sentry.setExtras({
+    doc_path: document?.fileInfo?.path,
+    doc_title: document?.metadata?.title,
+    doc_url: document?.url,
+  });
```

Comment:
From Sentry's docs, it seems like `setExtra` (and therefore, I assume, `setExtras` also) is deprecated in favour of structured contexts: https://docs.sentry.io/platforms/node/enriching-events/context/#additional-data, can we use `setContext` instead? https://docs.sentry.io/platforms/node/enriching-events/context/#structured-context